Makefile

```diff
@@ -1,44 +1,11 @@
+ROOT_DIR := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+CLEAN_PATTERNS := $(shell awk '/^# Clean$$/{f=1; next} /^# End Clean$$/{f=0} f' .gitignore)
 
 .PHONY: clean
-clean: clean-build clean-pyc clean-test clean-docs
-
-.PHONY: clean-build
-clean-build:
-	rm -fr build/
-	rm -fr dist/
-	rm -fr .eggs/
-	rm -fr deployments/build/
-	rm -fr deployments/Dockerfiles/open_aea/packages
-	rm -fr pip-wheel-metadata
-	find packages -name '*.egg-info' -exec rm -fr {} +
-	find packages -name '*.egg' -exec rm -fr {} +
-	find packages -name '*.svn' -exec rm -fr {} +
-	find packages -name '*.db' -exec rm -fr {} +
-	rm -fr .idea .history
-	rm -fr venv
-
-.PHONY: clean-docs
-clean-docs:
-	rm -fr site/
-
-.PHONY: clean-pyc
-clean-pyc:
-	find packages -name '*.pyc' -exec rm -f {} +
-	find packages -name '*.pyo' -exec rm -f {} +
-	find packages -name '__pycache__' -exec rm -fr {} +
-
-.PHONY: clean-test
-clean-test:
-	rm -fr .tox/
-	rm -f .coverage
-	find packages -name ".coverage*" -not -name ".coveragerc" -exec rm -fr "{}" \;
-	rm -fr coverage.xml
-	rm -fr htmlcov/
-	rm -fr .hypothesis
-	rm -fr .pytest_cache
-	rm -fr .mypy_cache/
-	find packages -name 'log.txt' -exec rm -fr {} +
-	find packages -name 'log.*.txt' -exec rm -fr {} +
```

Comment:
did you know this command does not work, because `packages` does not exist? So it does not clean

Reply:
Neither will this as shell is deprecated in 2.01 of poetry

Reply:
Packages exists after running make install which i believe is in the instructions for dev contributing?